charts/partials/templates/_trace.tpl

```diff
@@ -3,3 +3,10 @@
 - -trace-collector=linkerd-collector.{{.Values.global.namespace}}.svc.{{.Values.global.clusterDomain}}:55678
 {{ end -}}
 {{- end }}
+
+{{ define "partials.linkerd.trace.proxy" -}}
```

Comment:
Can we give this a more descriptive name like "configureControlPlaneTracing" or something like that?

Reply:
named it `partials.controlPlaneTracing.proxy` to signify that it's changing the proxy side of tracing variables.

Reply:
I don't want to block this over a naming bikeshed but this name is still pretty confusing to me.  This partial is a bit unusual in that it doesn't actually include any renderable template; instead it just sets some variables.  So when I first saw this, I was confused, thinking that it was rendering the proxy container or a control plane tracing version of the proxy container, or something like that.

I'd prefer a name that contains a word like "configure" or "set" or something else to indicate that this partial just sets values rather than rendering anything directly.